proto/src/determined/experiment/v1/experiment.proto

```diff
@@ -151,6 +151,8 @@ message Experiment {
   bool unmanaged = 40;
   // Time in seconds which experiment ran or has been running.
   optional int32 duration = 41;
+  // Size of model definition file, for unmanaged experiments this should be 0.
+  optional int32 model_definition_size = 45;
```

Comment:
Is there a reason for picking 45? I see 42 is taken?

Reply:
Just merged master, we have 43 and 44 to be external experiment/trial id